<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPUAlgorithm</class>
 <widget class="QDialog" name="CPUAlgorithm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run CPU HS Analysis</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="LayoutDialog">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>349</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="imgBox">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblEndmemberLocations">
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblOutputLog">
     <property name="text">
      <string>Started .</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Desired Regions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinDesiredRegions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Recursive Levels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinLevels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Spectral Clustering Weight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spinSpclustWeight">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>No of Clusters in intermediate levels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinNClustersIntermediate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>16</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Threads per Block</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinThreadsPerBlock">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="singleStep">
          <number>4</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="buttonsLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="5" column="1">
        <widget class="QPushButton" name="btnRHSEGStep">
         <property name="text">
          <string>RHSEG Step</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btnHSWOStep">
         <property name="text">
          <string>HSWO Step</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnRun">
         <property name="text">
          <string>Run HSEG</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnLoadImage">
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnRHSEG">
         <property name="text">
          <string>Run RHSEG</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnZoomIn">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btnZoomOut">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="btnRunCUDA">
         <property name="text">
          <string>Run GPU HSEG</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btnGPURHSEG">
         <property name="text">
          <string>Run GPU RHSEG</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="btnTest1">
         <property name="text">
          <string>Test1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="btnTest2">
         <property name="text">
          <string>Test2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>spinDesiredRegions</tabstop>
  <tabstop>btnLoadImage</tabstop>
  <tabstop>btnRun</tabstop>
  <tabstop>btnHSWOStep</tabstop>
  <tabstop>btnRHSEG</tabstop>
  <tabstop>btnRHSEGStep</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnRHSEG</sender>
   <signal>clicked()</signal>
   <receiver>CPUAlgorithm</receiver>
   <slot>onClicked_btnRHSEG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClicked_btnRHSEG()</slot>
 </slots>
</ui>
